strongbox-web-core/src/main/resources/views/directoryListing.twig.html

```diff
@@ -45,7 +46,7 @@ <h1>Index of {{ currentUrl }}/</h1>
 
         {% for directory in directories %}
         <tr>
-            <td><a href="{{ currentUrl }}/{{ directory.name }}/">{{ directory.name }}</a></td>
+            <td><a href="{{ currentUrl }}/{{ directory.name }}/">{{ directory.name }}/</a></td>
```

Comment:
do we need `{{ currentUrl }}` prefix it in case of `<base />` tag?

Reply:
We don't need it unless the url is going to another base (like the downloadl ink). Thanks for pointing it out! :)